playbooks/common/openshift-etcd/migrate.yml

```diff
@@ -93,17 +132,19 @@
   hosts: oo_masters_to_config
   roles:
   - role: etcd_migrate
+    r_etcd_migrate_action: add_ttls
```

Comment:
I can do that. I was trying to minimize the number of play transitions but I like what you're suggesting better.